src/event-helper.js

```diff
@@ -156,6 +159,15 @@ export function isLoaded(eleOrWindow) {
   );
 }
 
+/**
+ * Whether the specified element/window has errored already.
+ * @param {!Element|!Window} eleOrWindow
+ * @return {boolean}
+ */
+function isErrored(eleOrWindow) {
+  return !!eleOrWindow[LOAD_FAILURE_PROPERTY];
```

Comment:
Can we check only the `error` attribute? That way when the media changes src, it'll clear the error out.

Reply:
The `error` attribute isn't set when the error originated from a HTMLSourceElement, and is `read-only`. :(

Reply:
Then let's mark the error as coming from a particular src. If the src changes, then we ignore the `LOAD_FAILURE_PROPERTY`.

Reply:
Unfortunately it's not that easy, the sources could be a list of src coming from different HTMLSourceElements.

The only solution I have in mind would be to compute a string that's a list of all the `src` the browser tried before triggering the error (either the media `src`, or all its sources `src`).

If you think that's too crazy, if you're not comfortable with this change at the AMP level, I believe we could solve this reliably at the AMP Story level instead where we could easily remove that error property before changing the sources.

Reply:
Can't amp-video do the same thing?  I feel like adding this to stories code should be a last resort here.

Reply:
My worry is anything else that changes the `src` (eg, amp-bind). If we don't clear the `LOAD_FAILURE_PROPERTY` there, too, then `loadPromise` will incorrectly fail for the media element.

Reply:
Thanks both for the comments.
I've been able to come up with an implementation that resolves Justin's concerns, using `HTMLMediaElement.currentSrc`.
PTAL